.github/workflows/conda_ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Verify functionality of conda deployment
+
+on:
+  # Run if anything changes related to tool setup
+  push:
+    paths: 'setup/**'
+  pull_request:
+    paths: 'setup/**'
+    
+jobs:
+  test-basic-tools:
+    name: Test basic tools
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.11"] # ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
any reason the other versions are not included?

Reply:
Just for quicker testing, I enabled them back.